app/assets/stylesheets/common/d-editor.scss

```diff
@@ -5,6 +5,8 @@
 
   &--rich-editor-enabled .d-editor-textarea-wrapper textarea.d-editor-input {
     font-family: var(--d-font-family--monospace);
+    font-size: var(--font-down-1);
+    font-variant-ligatures: none;
```

Comment:
We don't need this right, since this ligatures rule is already added when jetbrains mono is used when rich editor is enabled?

Reply:
sadly we need this, cause textarea for whatever crazy reason removes this :cry:  so we need to re-apply